server/chat-commands/core.ts

```diff
@@ -965,6 +965,24 @@ export const commands: ChatCommands = {
 	},
 	importinputloghelp: [`/importinputlog [inputlog] - Starts a battle with a given inputlog. Requires: + % @ &`],
 
+	async reseedbattle(target, room, user) {
+		room = this.requireRoom();
+		if (!room.battle) return this.errorReply(`This command can only be used in battle.`);
+		this.checkCan('editbattle');
```

Comment:
It turns out editbattle uses the `forcewin` permission; it doesn't have its own permission.